### PR TITLE
Desuger ModuleStore to es5; move it up in the build file.

### PR DESCRIPTION
**Pull request by johnjbarton**
This should allow files below ModuleStore in the build file
to be loaded as modules.

See #1628

### CODE REVIEW

File: src/runtime/ModuleStore.js [hunk 1]
```diff
@@ -31,43 +31,45 @@
...
-        this.stack = '';
-    }
+    if (!(cause instanceof ModuleEvaluationError) && cause.stack)
+      this.stack = this.stripStack(cause.stack);
+    else
+      this.stack = '';
+  }
+
+  ModuleEvaluationError.prototype = {
+    __proto__: Error.prototype,
```
Comment (arv) [inline src/runtime/ModuleStore.js]: This does not work in IE10 :'(

I'm not sure it matters though?

File: src/runtime/ModuleStore.js [hunk 2]
```diff
@@ -110,55 +111,57 @@
...
-              evaled = evaled.concat(afterLines(lines, line));
-              evaled.push('= = = = = = = = =');
-            } else {
-              evaled.push(frame);
-            }
-          });
-          ex.stack = evaled.join('\n');
-        }
+    UncoatedModuleInstantiator.prototype = {
+      __proto__: UncoatedModuleEntry.prototype,
```
Comment (arv) [inline src/runtime/ModuleStore.js]: same here

**Comment by arv** [comment]
I assume the runtime functions have just been reindented (and iife has been removed)?

**Comment by arv** [comment]
Very nice. Once this is in we should probably use export statements.

File: src/runtime/ModuleStore.js (hunk 2 above)
Comment (johnjbarton) [inline src/runtime/ModuleStore.js]: done

File: src/runtime/ModuleStore.js (hunk 1 above)
Comment (johnjbarton) [inline src/runtime/ModuleStore.js]: done

**Comment by johnjbarton** [comment]
PTAL

**Comment by arv** [comment]
LGTM

**Pull request closed without merging**